### PR TITLE
[zh-cn] Fix some invalid links

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Fix for:
- [内容改进建议] in https://kubernetes.io/zh-cn/docs/contribute/localization/
- [应用程序与服务连接] in https://kubernetes.io/zh-cn/docs/concepts/services-networking/cluster-ip-allocation/
- [5] in https://kubernetes.io/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/
- [6] in https://kubernetes.io/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/

### CODE REVIEW

**Review by ydFu — APPROVED** [review]
/lgtm

**Comment by windsonsea** [comment]
Thanks.
/approve

**Comment by howieyuen** [comment]
`static/_redirects` cannot add localized links
/hold

**Comment by howieyuen** [comment]
@tengqm, shall we update these links in localized page ?

File: static/_redirects [hunk 1]
```diff
@@ -321,12 +322,14 @@
 /docs/tutorials/kubernetes-basics/expose-intro/     /docs/tutorials/kubernetes-basics/expose/expose-intro/ 301
 /docs/tutorials/kubernetes-basics/scale-interactive/     /docs/tutorials/kubernetes-basics/scale/scale-interactive/ 301
 /docs/tutorials/kubernetes-basics/scale-intro/     /docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
+/zh-cn/docs/tutorials/kubernetes-basics/scale-intro/     /zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
```
Comment (tengqm) [inline static/_redirects:325]: why?

Comment (fenggw-fnst) [reply static/_redirects:325]: Thanks for review. 
Please check this page: https://kubernetes.io/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/
The invalid link is: 模块 **[5]** 和 6 讨论了如何扩展和更新 Deployments。

File: static/_redirects [hunk 2]
```diff
@@ -77,6 +77,7 @@
 /id/docs/concepts/jobs/run-to-completion-finite-workloads/      /id/docs/concepts/workloads/controllers/job/ 301
 /docs/concepts/nodes/node/     /docs/concepts/architecture/nodes/ 301
 /docs/concepts/services-networking/connect-applications-service/    /docs/tutorials/services/connect-applications-service/   301
+/zh-cn/docs/concepts/services-networking/connect-applications-service/    /zh-cn/docs/tutorials/services/connect-applications-service/   301
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply static/_redirects:80]: Yes, in this page: https://kubernetes.io/zh-cn/docs/concepts/services-networking/cluster-ip-allocation/
The invalid link is: 阅读 **[应用程序与服务连接]**

File: static/_redirects [hunk 3]
```diff
@@ -321,12 +322,14 @@
...
 /docs/tutorials/kubernetes-basics/scale-interactive/     /docs/tutorials/kubernetes-basics/scale/scale-interactive/ 301
 /docs/tutorials/kubernetes-basics/scale-intro/     /docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
+/zh-cn/docs/tutorials/kubernetes-basics/scale-intro/     /zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
 /docs/tutorials/clusters/apparmor/     /docs/tutorials/security/apparmor/   301
 /docs/tutorials/clusters/seccomp/      /docs/tutorials/security/seccomp/    301
 /ja/docs/tutorials/kubernetes-basics/scale-intro/     /ja/docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
 /ko/docs/tutorials/kubernetes-basics/scale-intro/     /ko/docs/tutorials/kubernetes-basics/scale/scale-intro/ 301
 /docs/tutorials/kubernetes-basics/update-interactive/    /docs/tutorials/kubernetes-basics/update/update-interactive/ 301
 /docs/tutorials/kubernetes-basics/update-intro/     /docs/tutorials/kubernetes-basics/update/update-intro/ 301
+/zh-cn/docs/tutorials/kubernetes-basics/update-intro/     /zh-cn/docs/tutorials/kubernetes-basics/update/update-intro/ 301
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply static/_redirects:332]: In this page: https://kubernetes.io/zh-cn/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/
The invalid link is: 模块 5 和 **[6]** 讨论了如何扩展和更新 Deployments。

[the remark this answers is not in the document]
Comment (tengqm) [reply static/_redirects:332]: OK, then please fix those files instead.

**Comment by tengqm** [comment]
Instead of adding redirection entries, we should strive to fix the links directly.

You can try run the following command to find out the (possible) invalid links in zh-cn localization:

```
./scripts/linkchecker.py -f 'content/zh-cn/docs/**/*.md'
```

**Comment by fenggw-fnst** [comment]
> Instead of adding redirection entries, we should strive to fix the links directly.

Understood. Thanks for your suggestion.

> You can try run the following command to find out the (possible) invalid links in zh-cn localization:
./scripts/linkchecker.py -f 'content/zh-cn/docs/**/*.md'

Yes, I did find these links through this script, and these should be all of the invalid ones that I have found.

File: static/_redirects (hunk 3 above)
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply static/_redirects]: Thanks, all done.

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Pull request merged**